.prettierrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
prettier's config setting indentation to 2 spaces. Other options here: https://prettier.io/docs/en/options.html